addon-test-support/helpers.js

```diff
@@ -168,42 +168,47 @@ export function keyEvent(selector, type, keyCode) {
 }
 
 /*
-  The find test helper uses `querySelectorAll` to search inside the test
+  The find test helper uses `querySelector` to search inside the test
   DOM (based on app configuration for the rootElement).
 
   Alternalively, a second argument may be passed which is an element as the
   DOM context to search within.
 
   @method find
-  @param {String|HTMLElement|NodeList} CSS selector to find one or more elements in the test DOM
+  @param {String} CSS selector to find one or more elements in the test DOM
   @param {HTMLElement} contextEl to query within, query from its contained DOM
-  @return {null|HTMLElement|NodeList} null, one element or a (non-live) list of element objects
+  @return {null|HTMLElement} null or an element
   @public
 */
 export function find(selector, contextEl) {
   let result;
-  if (selector instanceof HTMLElement || selector instanceof NodeList) {
-    result = selector;
-  } else if (contextEl instanceof HTMLElement) {
-    result = contextEl.querySelectorAll(selector);
+  if (contextEl instanceof HTMLElement) {
+    result = contextEl.querySelector(selector);
   } else if (Object.prototype.toString.call(selector) === "[object String]") {
```

Comment:
I know this is unrelated to the PR at hand, but I think this can use `else if (typeof selector === 'string') {` instead (which is much easier for me to grok when reading).